src/box.jl

```diff
@@ -31,8 +31,8 @@ end
 
 function normal(x::SVector{N}, b::Box{N}) where {N}
     d = b.p * (x - b.c)
-    (m,i) = findmin(abs.(abs.(d) - b.r))
-    return SVector{N}(b.p[i,:]) * sign(d[i])
+    ~, i = findmin(abs.(abs.(d) - b.r))
```

Comment:
Don't use `~` as a variable name; it could easily have a special meaning in a future Julia version